src/routes/app/wallets/index.tsx

```diff
@@ -314,6 +314,7 @@ export default component$(() => {
   const handleAddWallet = $(async () => {
     console.log("ADDING WALLET...");
     isAddWalletModalOpen.value = false;
+    console.log("addwalletFormStore", addWalletFormStore);
```

Comment:
Do we need this log here?

Reply:
Used mailny for debugging, since we don't have any logger yet.
Removed that